apps/next/components/FAQ/index.tsx

```diff
@@ -149,7 +151,7 @@ export default function FAQ() {
   return (
     <>
       <div className="flex-col w-full mt-6 mx-auto lg:flex lg:flex-row lg:w-full">
-        <div className="flex-col bg-gray-800 px-4 py-16 w-full rounded-md lg:px-32">
+        <div className="flex-col bg-gray-800 px-4 py-16 w-full rounded-md lg:px-[70px]">
```

Comment:
70 pixels is specific. Have you tested this on different large resolutions, different browsers, etc?

Reply:
px-[var] is the vertical padding on this container. I realized the space between the FAQ container with all of the questions/answers and the outer container was very very wide so I thinned it down so the FAQ column was a bit more prominent on the FAQ page. I used my best judgement and thought 70px on larger viewports (essentially anything that's larger than mobile) looked best. Otherwise it was px-32 which is also a static size, but much more padding than 70px.

NOTE: This change only affects the container if the viewport is larger than mobile.